bin/stub.rb.erb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.unshift('<%= rspec_bash_library_path_for_template %>')
+
+require 'rspec/bash'
+require 'pathname'
+require 'yaml'
+
+include Rspec::Bash
+
+command = File.basename(__FILE__)
+folder = File.dirname(__FILE__)
+
+call_log_path = Pathname.new(folder).join("#{command}_calls.yml")
```

Comment:
I think `Pathname.new(File.join(folder, "#{command}_calls.yml"))` is better here, but I don't remember why. I'll get back to you on this...